add new pattern for dates pipeline (#74)
aphp/edsnlp: changelog.md

# Changelog

> ## Pending release
> ### Added
> - Add new pattern to pipeline `dates` to identify cases where only the month is mentioned.

Reviewer: ```suggestion
## Pending release

### Changed

- Add new pattern to pipeline `esd.dates` to identify cases where only the month is mentioned.
```

# Changelog

## Pending release

> ### Fixed

Reviewer: ```suggestion
### Added
```

# Changelog

> ## Pending release

Reviewer: ```suggestion

## Pending release
```
